pub-sub/README.md

```diff
@@ -98,19 +98,19 @@ The Dapr CLI provides a mechanism to publish messages for testing purposes. Now,
   * Linux/Mac:
 
 ```bash
-dapr publish --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
+dapr publish --publish-app-id python-subscriber --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
```

Comment:
`publish-app-id` is supposed to be used with the publisher app and not the subscriber app. Can we run the react form app which is the publisher in this case, show the CLI way of publishing events and the show the GUI?

Reply:
/cc @msfussell

Reply:
LGTM

Reply:
Please, see how it reads now with the latest version.